src/client.js

```diff
@@ -175,6 +175,68 @@ function keyFromRecoverySession(session, decryptionKey) {
  * @param {boolean} [opts.fallbackICEServerAllowed]
  * Optional. Whether to allow a fallback ICE server should be used for negotiating a
  * WebRTC connection if the homeserver doesn't provide any servers. Defaults to false.
+ *
+ * @param {object} opts.cryptoCallbacks Optional. Callbacks for crypto and cross-signing.
+ *
+ * @param {function} [opts.cryptoCallbacks.getCrossSigningKey]
```

Comment:
Yeah, although probably just marking them unstable in the docs is enough? Since clients that don't want to engage with cross-signing don't have to, and nothing will change (unlike unstableClientRelationAggregation which is changing how existing API works).

Reply:
Okay, that seems reasonable to me. 😄